app/src/main/res/layout/feedinfo.xml

```diff
@@ -159,7 +159,38 @@
                 android:enabled="false"
                 android:textColor="?android:attr/textColorPrimary"
                 tools:background="@android:color/holo_red_light"
-                android:checked="false"/>
+                android:checked="false" />
+
+            <android.support.v7.widget.GridLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                app:columnCount="2"
+                app:rowCount="1">
+
+                <TextView
+                    android:id="@+id/txtvFeedAutoDelete"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/auto_delete_label"
+                    app:layout_row="0"
+                    app:layout_column="0"
+                    app:layout_gravity="center_vertical"
+                    android:layout_marginRight="10dp" />
+
+                <Spinner
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:id="@+id/spnAutoDelete"
+                    android:entries="@array/spnAutoDeleteItems"
+                    android:layout_marginTop="8dp"
+                    app:layout_row="0"
+                    app:layout_column="1"
+                    android:spinnerMode="dropdown"
+                    app:layout_gravity="center"
+                    android:dropDownWidth="wrap_content"
+                    android:clickable="false" />
```

Comment:
When I test this PR on a Virtual "Nexus 5 API 21 x86" it definitely opens the spinner.
I've checked all the Spinner combinations (Global/Yes/No) against the Auto Delete Settings both in the checked and unchecked position.

Spinner	Settings	Result
Global	Checked	Deleted
Yes		Checked	Deleted
No		Checked	Not Deleted
Global	UnChecked	Not Deleted
Yes		UnChecked	Deleted
No		UnChecked	Not Deleted

As I wrote earlier, this is the first Android project I'm involved in, and most of the coding is based on cloning existing style in AntennaPod or on googling (thanks stackexchange & github...).
I think that android:clickable doesn't matter here because I use OnItemSelectedListener().